### PR TITLE
Use cluster_* variables in kubernetes.yml config

### PR DESCRIPTION
**Pull request by markgoddard**
This brings this file into line with swarm-sip.yml. Also update the test-*.yml
configs to use the cluster_* variables.

### CODE REVIEW

File: config/kubernetes.yml [hunk 1]
```diff
@@ -5,11 +5,6 @@ bdn_name: p3-bdn
 lln_name: p3-lln
 
 # Kubernetes
-os_container_infra_user: fedora
-os_container_infra_state: present
-os_container_infra_cluster_name: kubernetes
-os_container_infra_cluster_template_name: k8s-fedora-atomic-27
-os_container_infra_keypair: default
 os_container_infra_master_count: 1
```
Comment (brtknr) [inline config/kubernetes.yml]: missed one

File: config/kubernetes.yml [hunk 2]
```diff
@@ -5,11 +5,6 @@ bdn_name: p3-bdn
...
 os_container_infra_node_count: 6
 os_container_infra_external_interface: "{{ ext_name }}"
```
Comment (brtknr) [inline config/kubernetes.yml]: and another one

File: config/kubernetes.yml [hunk 3]
```diff
@@ -5,11 +5,6 @@ bdn_name: p3-bdn
 lln_name: p3-lln
 
 # Kubernetes
-os_container_infra_user: fedora
-os_container_infra_state: present
-os_container_infra_cluster_name: kubernetes
-os_container_infra_cluster_template_name: k8s-fedora-atomic-27
-os_container_infra_keypair: default
 os_container_infra_master_count: 1
 os_container_infra_node_count: 6
```
Comment (brtknr) [inline config/kubernetes.yml]: and another one

**Comment by markgoddard** [comment]
> is this still WIP? I dont see equivalent changes to container-infra.yml playbook.

As discussed in person - no it's ready to go.

**Pull request merged**